armi/tests/tutorials/param_sweep.ipynb

```diff
@@ -0,0 +1,314 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## A simple one-dimensional parameter sweep"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "builder = suiteBuilder.SeparateEffectsSuiteBuilder(case)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each degree of freedom is defined by a `InputModifier` and a range of values. ARMI contains a few basic `InputModifier` for simple things (like changing settings), and for design-specific param sweeps you can make your own design-specific modifiers. \n",
+    "\n",
+    "The simplest form of parameter sweep just adjust settings. For example, we could adjust the reactor power from 10 MW to 100 MW in a few steps. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "powers = np.linspace(10,100,4)\n",
+    "print(f\"Building power modifiers with powers: {powers}\")\n",
+    "powerModifications = [suiteBuilder.SettingsModifier('power', mw*1e6) for mw in powers]\n",
+    "builder.addDegreeOfFreedom(powerModifications)\n",
+    "print(f\"There are {len(builder.modifierSets)} cases in this suite so far.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can build the suite. The `Suite` object itself can write input files or just run on the local computer with `suite.run`.\n",
+    "\n",
+    "The suite will generate copies of the base case with the power modified across the defined range. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "suite = builder.buildSuite()\n",
+    "suite.echoConfiguration()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On the other hand, if you want to write inputs and then submit them all to a high-performance computer, you can do that too with `suite.writeInputs()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "suite.writeInputs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can now see that perturbed input files have been produced in the `case-suite` folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!grep -R \"power:\" case-suite/*"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "To submit this suite to a computer cluster, one would run a command simliar to ``python -m tparmi submit-suite`` from the ``case-suite`` folder. \n",
+    "\n",
+    ".. note:: This requires a compatible HPC plugin for your HPC environment in your application."
```

Comment:
This might be a bit confusing to a user that's never seen our plugin before. Maybe provide more context, or simply recommend "submitting many ARMI `run` commands using whichever job queuing system is available in your HPC environment."

Reply:
good point.